stdlib/public/TensorFlow/Ops.swift

```diff
@@ -1307,7 +1307,9 @@ public extension Tensor {
 // Normalization
 //===----------------------------------------------------------------------===//
 
-public extension Tensor where Scalar : BinaryFloatingPoint {
+public extension Tensor where Scalar : BinaryFloatingPoint,
+                              Scalar : Differentiable,
+                              Scalar.CotangentVector == Scalar {
```

Comment:
Could you briefly explain why the extra conditions on `Scalar` are necessary? Thanks 🙂

Reply:
I'll add a comment.